coral/src/app/features/topics/details/subscriptions/TopicSubscriptions.tsx

```diff
@@ -43,8 +40,9 @@ const isSubscriptionsOption = (value: string): value is SubscriptionOptions => {
 
 const TopicSubscriptions = () => {
   const navigate = useNavigate();
-  // @ TODO get environment from useTopicDetails too when it is implemented2
-  const { topicName } = useTopicDetails();
+  const { topicOverview, environmentId } = useTopicDetails();
+
+  const topicName = topicOverview.topicInfoList[0].topicName;
```

Comment:
Could we have a `topicName` entry in `useTopicDetails`, even if it's slightly redundant with the `topicOverview` data? It feels a little weird to do this complex object delving every time we need it :o

Or maybe `topicName` could be part of the response payload of `getTopicOverview` ^^

Reply:
It is a bit weird that it's complex, but I wanted to avoid that we pass a topicName while a topic may not exist. I think it would be nice to have the name on the `TopicOverview` object as a direct property though! @aindriu-aiven  would that be possible (maybe not right now, but for the next release)?

Reply:
What would be the problem if we pass a `topicName` as `undefined` or empty string when a topic does not exist? :o

Reply:
Hm 🤔 To be clear, what I'm proposing is something like this:
```tsx
        <Outlet context={{ environmentId, topicOverview, topicName: topicOverview.topicInfoList[0].topicName }} />
```

So `topicOverview` is still the source of truth, we just implement a nicety in the context ^^

Reply:
💡  didn't think of that! 🤦‍♀️  Yes,  I can update that!

Reply:
updated!